Modules/_tkinter.c

```diff
@@ -3207,17 +3207,11 @@ PyInit__tkinter(void)
     if (m == NULL)
         return NULL;
 
-    o = PyErr_NewException("_tkinter.TclError", NULL, NULL);
-    if (o == NULL) {
+    Tkinter_TclError = PyErr_NewException("_tkinter.TclError", NULL, NULL);
+    if (PyModule_AddObjectRef(m, "TclError", Tkinter_TclError)) {
         Py_DECREF(m);
```

Comment:
Note for later: i prefer "exec" functions which returns -1 on error, and the caller is responsible to manage the module refcount.